roadmap/implementers-guide/src/node/utility/availability-store.md

```diff
@@ -9,19 +9,13 @@ The two data types:
 
 For each of these data we have pruning rules that determine how long we need to keep that data available.
 
-PoV hypothetically only need to be kept around until the block where the data was made fully available is finalized. However, disputes can revert finality, so we need to be a bit more conservative. We should keep the PoV until a block that finalized availability of it has been finalized for 1 day.
+PoV hypothetically only need to be kept around until the block where the data was made fully available is finalized. However, disputes can revert finality, so we need to be a bit more conservative and we add a delay. We should keep the PoV until a block that finalized availability of it has been finalized for 1 day + 1 hour.
 
-> TODO: arbitrary, but extracting `acceptance_period` is kind of hard here...
-
-Availability chunks need to be kept available until the dispute period for the corresponding candidate has ended. We can accomplish this by using the same criterion as the above, plus a delay. This gives us a pruning condition of the block finalizing availability of the chunk being final for 1 day + 1 hour.
-
-> TODO: again, concrete acceptance-period would be nicer here, but complicates things
+Availability chunks need to be kept available until the dispute period for the corresponding candidate has ended. We can accomplish this by using the same criterion as the above. This gives us a pruning condition of the block finalizing availability of the chunk being final for 1 day + 1 hour.
 
 There is also the case where a validator commits to make a PoV available, but the corresponding candidate is never backed. In this case, we keep the PoV available for 1 hour.
 
-> TODO: ideally would be an upper bound on how far back contextual execution is OK.
-
-There may be multiple competing blocks all ending the availability phase for a particular candidate. Until (and slightly beyond) finality, it will be unclear which of those is actually the canonical chain, so the pruning records for PoVs and Availability chunks should keep track of all such blocks.
+There may be multiple competing blocks all ending the availability phase for a particular candidate. Until finality, it will be unclear which of those is actually the canonical chain, so the pruning records for PoVs and Availability chunks should keep track of all such blocks.
```

Comment:
We should keep the unfinalized chain somewhat longer for possible remote disputes.  We've two methods for doing this, either actually keep the data for a while, although not sure how long, or else later add some path for disputers to rerun distribution.  (ugh!)  We could just add some relatively short delay for the time being.

Also, nodes should never deletes anything for which they sign a dispute, maybe even making some backup outside the normal database structure.  We could postpone doing this however.